src/ring_theory/localization/norm.lean

```diff
@@ -0,0 +1,50 @@
+/-
+Copyright (c) 2023 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen
+-/
+
+import ring_theory.localization.module
+import ring_theory.norm
+
+/-!
+
+# Field/algebra norm and localization
+
+This file contains results on the combination of `algebra.norm` and `is_localization`.
+
+## Main results
+
+ * `algebra.norm_localization`: let `S` be an extension of `R` and `Rₘ Sₘ` be localizations at `M`
+  of `R S` respectively. Then the norm of `a : Sₘ` over `Rₘ` is the norm of `a : S` over `R`.
+
+## Tags
+
+field norm, algebra norm, localization
+
+-/
+
+open_locale non_zero_divisors
+
+variables (R : Type*) {S : Type*} [comm_ring R] [comm_ring S] [algebra R S]
+variables {Rₘ Sₘ : Type*} [comm_ring Rₘ] [algebra R Rₘ] [comm_ring Sₘ] [algebra S Sₘ]
+variables {M : submonoid R}
+variables [is_localization M Rₘ] [is_localization (algebra.algebra_map_submonoid S M) Sₘ]
+variables [algebra Rₘ Sₘ] [algebra R Sₘ] [is_scalar_tower R Rₘ Sₘ] [is_scalar_tower R S Sₘ]
+
+/-- Let `S` be an extension of `R` and `Rₘ Sₘ` be localizations at `M` of `R S` respectively.
+Then the norm of `a : Sₘ` over `Rₘ` is the norm of `a : S` over `R`. -/
+lemma algebra.norm_localization [nontrivial R] [module.free R S] [module.finite R S]
```

Comment:
This should (trivially) hold over the trivial ring unless I am missing something stupid.